dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -761,6 +761,9 @@
 				Update="@(ResolvedFileToPublish)"
 				RelativePath="$([MSBuild]::MakeRelative($(MSBuildProjectDirectory)$(PublishDir),$(_AssemblyPublishDir)))\%(Filename)%(Extension)"
 				Condition="'$(_PlatformName)' != 'macOS' And '$(InvariantGlobalization)' != 'true' And '%(Filename)%(Extension)' == 'icudt.dat'" />
+
+			<!-- Remove the libxamarin-*.dylib files we don't want -->
+			<ResolvedFileToPublish Remove="@(ResolvedFileToPublish)" Condition="'%(Extension)' == '.dylib' And '%(Filename)%(Extension)' != '$(_LibXamarinName)' And $([System.String]::new('%(Filename)').StartsWith('libxamarin-dotnet-'))" />
```

Comment:
`StartsWith` should use `, StringComparison.Ordinal`

Reply:
I am 100% that you have told me this 1000000 times 🤣